engine/src/main/java/com/datatorrent/stram/api/OperatorDeployInfo.java

```diff
@@ -314,6 +319,10 @@ public void sendMetrics(Collection<String> metricNames)
   public AttributeMap contextAttributes;
 
   /**
+   * Operator deploy reason groupId (event group id)
+   */
+  public long deployGroupId;
```

Comment:
How that ensures that groupId is unique? What if two containers fail at the same timestamp? Not a probable event in case of a small number of containers, but more likely to happen when the number of containers is large and they all fail for example due to errors on hdfs side?